tests/test_fx/test_meta/test_aten.py

```diff
@@ -0,0 +1,88 @@
+from typing import Any, Callable, Union
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from colossalai.fx.profiler import MetaTensor
+
+import pytest
+
+try:
+    meta_lib = torch.library.Library("aten", "IMPL", "Meta")
+    incompatible = False    # version > 1.12.0
```

Comment:
Use constant for this value  i.e. `INCOMPATIBLE = False`

Reply:
Solved.